jenkins/bin/reload-jobs.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# create and update all jobs
+set -e
+
+reload() {
+
+    # the job-reloading-job needs to update itself, but note that changes
+    # won't be picked up in this pass because we're already running the
+    # job. kick the build off manually if you need it immediately, otherwise
+    # wait until the next pass
+    xmlstarlet \
+        ed \
+        -u '//project/scm/userRemoteConfigs/hudson.plugins.git.UserRemoteConfig/url' \
+        -v "${GITHUB_JOBS_REPO}" \
+        -u '//project/scm/branches/hudson.plugins.git.BranchSpec/name' \
+        -v "${GITHUB_JOBS_SPEC:-master}" \
+        /usr/share/jenkins/templates/jenkins-jobs.config.xml \
+        > "${JENKINS_HOME}/jobs/jenkins-jobs/config.xml"
+
+    for job in ${WORKSPACE}/jenkins/jobs/*; do
+
+        local jobname=$(basename ${job})
+        local config=${WORKSPACE}/jenkins/jobs/${jobname}/config.xml
+
+        # TODO: we probably want a dedicated user for this rather than 'tgross'
+        # create a new job for each directory under workspace/jobs
+        curl -XPOST -s -o /dev/null \
+             -d @"${config}" \
+             --user "tgross:${GITHUB_API_TOKEN}" \
```

Comment:
Fixed in https://github.com/autopilotpattern/jenkins/pull/14/commits/20427e99a6842a1ac1b70a214729e0bf57026908